### ISSUE TITLE
SWID Tag ID not preserved during clone operation

### ISSUE THREAD

**Issue by canon-cmi-taylor-jakobson** [DependencyTrack/dependency-track#4478]
### Current Behavior

When cloning a projects, the SWID Tag ID is removed from all components.
This causes an issue when a newer version of the SBOM is uploaded to the project, as the component uuid changes and we lose analysis information from the original project even though the component in the SBOM hasn't changed.

### Steps to Reproduce

1. Create a project
2. Upload the attached SBOM
3. Clone the project
4. Compare the SWID Tag ID field in the new project to the original project

[swid-example.cdx.json](https://github.com/user-attachments/files/18200239/swid-example.cdx.json)

### Expected Behavior

The SWID Tag ID is retained, ensuring that cloning a project and re-uploading the SBOM does not result in the loss of analysis data.

### Dependency-Track Version

4.12.0

### Dependency-Track Distribution

Container Image

### Database Server

Microsoft SQL Server

### Browser

Google Chrome

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this defect was already reported

**Comment by nscuro** [comment]
Yup, CPE and PURL are carried over, but the SWID Tag ID is not: https://github.com/DependencyTrack/dependency-track/blob/45982a2e1a12a0b0de17f955b856b27008fee02a/src/main/java/org/dependencytrack/persistence/ComponentQueryManager.java#L370-L372

Thanks for reporting.
